simulation_parameters/common/input_options.json

```diff
@@ -380,6 +380,10 @@
       {
         "InputName": "g_toggle_gui",
         "Name": "TOGGLE_HUD_HIDE"
+      },
+      {
+        "InputName": "g_toggle_speed_mode",
+        "Name": "TOGGLE_FAST_MODE"
```

Comment:
I think this doesn't fully fit in the miscellaneous section... I'd rather put this in `COMMON_ABILITIES`. This doesn't fully fit there either but I think it would be better.